library/dd-sdk-android-trace/src/test/kotlin/com/datadog/android/utils/forge/SpanEventForgeryFactory.kt

```diff
@@ -77,20 +76,20 @@ internal class SpanEventForgeryFactory : ForgeryFactory<SpanEvent> {
         )
     }
 
-    fun Forge.exhaustiveMetrics(): MutableMap<String, Number> {
+    private fun Forge.exhaustiveMetrics(): MutableMap<String, Number> {
         return listOf(
             aLong(),
             anInt(),
             aFloat(),
             aDouble()
         ).map { anAlphabeticalString() to it as Number }
-            .toMutableMap()
+            .toMap(mutableMapOf())
```

Comment:
what is the advantage ?

Reply:
no advantage, the original method is just private in the main module, so it was easier to to the replacement than to share it.